examples/example-gateway/clients/bar/client-config.json

```diff
@@ -18,6 +18,7 @@
 			"ArgWithManyQueryParams": "Bar::argWithManyQueryParams",
 			"ArgWithParams": "Bar::argWithParams",
 			"Hello": "Bar::helloWorld",
+			"TestUrl": "Bar::testUrl",
```

Comment:
The user should just provide `TestURL` to make linter pass in the first place.

Reply:
@ChuntaoLu it's more like failing our server side proxy method names instead of affecting clients. So my suggestion is to fix it in our side instead of pushing it down to clients. 
Reusing the `commonInitialisms` to simplify the process.